docs/content/FilteredSearch.md

```diff
@@ -15,14 +15,14 @@ The FilteredSearch component helps style a Dropdown and a TextInput side-by-side
   <ActionMenu>
     <ActionMenu.Button as="summary">Filter</ActionMenu.Button>
     <ActionMenu.Overlay>
-      <ActionList direction="sw">
```

Comment:
Action List doesn't seem to have a direction prop ([ref](https://primer.style/react/deprecated/Dropdown#deprecation))

Reply:
Good catch!